Text_Mining.py

```diff
@@ -0,0 +1,161 @@
+import requests
```

Comment:
Header comment here with an information about the author of this code (you) and a brief description of this file would be helpful.